internal/common/baseclient/basejudge.go

```diff
@@ -9,6 +9,18 @@ import (
 type BaseJudge struct {
 }
 
+// GetRuleViolationSeverity returns a custom map of named rules and how severe the sanction should be for transgressing them
+// If a rule is not named here, the default sanction value added is 1
+func (j *BaseJudge) GetRuleViolationSeverity() map[string]roles.IIGOSanctionScore {
+	return map[string]roles.IIGOSanctionScore{}
```

Comment:
were rules not changed to be an enum. 

After reading the rest i assume you expect the client to call .string on the rule.
Would it be possible to get an example of how a client would interact with these features. Potentially in a future PR after these 4 are complete

Reply:
Right now whether rules are enum's is an open question since design haven't yet closed the door to invented rules. Will update if that changes.

Reply:
Yes I'll include examples of use in a future PR. In fact, could we have like a "Demo" client set up for the infra teams to demo their features (rather than in base client which we'd prefer to keep plain and stable?) @NGovani @lhl2617